sidecar/internal/cmd/start.go

```diff
@@ -63,9 +63,7 @@ func Start(_ *cobra.Command, _ []string) {
 	}()
 
 	slog.Info(fmt.Sprintf("SSV sidecar started, serving on port %d", PortFlag))
-	for {
-		err := <-errs
-		slog.Error("error running daemon", err)
-		os.Exit(1)
-	}
+	err = <-errs
+	slog.Error("error running daemon", "err", err)
+	os.Exit(1)
```

Comment:
So, no graceful exit option in case of no errors?

Reply:
Have now checked for `ErrServerClosed`